packages/verified-fetch/src/verified-fetch.ts

```diff
@@ -277,9 +302,14 @@ export class VerifiedFetch {
 
     response.headers.set('content-type', accept)
 
+    if (ipfsRoots != null) {
+      response.headers.set('X-Ipfs-Roots', ipfsRoots.map(cid => cid.toV1().toString()).join(',')) // https://specs.ipfs.tech/http-gateways/path-gateway/#x-ipfs-roots-response-header
+    }
+
     return response
   }
 
+  // eslint-disable-next-line complexity
```

Comment:
Disabling linting is rarely a good idea - can this method be decomposed instead?

Reply:
yea we definitely need to break some stuff out. i'll take a peek

Reply:
I was able to remove the lint rule disabling by converting to `options?.signal?.throwIfAborted()`. will open an issue to track a fix so someone else isn't hit with this whenever editing those functions.

Reply:
https://github.com/ipfs/helia-verified-fetch/issues/43